src/python/foglamp/core/service_registry/service_registry.py

```diff
@@ -133,18 +146,23 @@
             "type": service._type,
             "address": service._address,
             "port": service._port,
+            "management_port": service._management_port,
```

Comment:
This does not conform to the conventions we use. It would be managementPort

Reply:
I believe, for input keys we are **snake_case**

Reply:
and this also aligns with python naming convention.